app/models/claim.rb

```diff
@@ -52,7 +52,8 @@ class Claim < ApplicationRecord
   belongs_to :eligibility, polymorphic: true
   accepts_nested_attributes_for :eligibility, update_only: true
 
-  belongs_to :payroll_run, optional: true
+  has_one :payment
+  has_one :payroll_run, through: :payment
```

Comment:
The first few commits of this PR do make use of it, because we take advantage of the fact that, before we add the presence validation for `award_amount` in a later commit, Rails automatically creates the payments if you assign the payroll run directly to the claim. That is, we have code like:

```ruby
create_list(:claim, 2, :approved, payroll_run: payroll_run)
```

Once we introduce the validation and hence start creating payments explicitly, we no longer make use of the association. So I think the easiest thing to do will be to start off _with_ the association, and then remove it in the commit where we introduce the validation.